glide.yaml

```diff
@@ -0,0 +1,28 @@
+package: github.com/coreos/locksmith
+import:
+- package: github.com/coreos/etcd
+  version: v2.2.5
+  subpackages:
+  - client
+  - pkg/pathutil
+  - pkg/types
+- package: github.com/coreos/go-systemd
+  version: e3e4f602334ec3e1c2cd68322a1284903c62f940
+  subpackages:
+  - dbus
+  - journal
+  - login1
+- package: github.com/coreos/pkg
+  version: 549bd7890e3564e9a2409d9700ea365005edad8d
+  subpackages:
+  - capnslog
+- package: github.com/godbus/dbus
+  version: 25a4b8ca48c64b3d631110fb74a210f56c401e0b
```

Comment:
I switched the one dependency which was already pointing at a tagged release, but the rest of them are pointing at commits which aren't tagged. I want to investigate possible dependency upgrades, but didn't want to as part of this pr, since upgrading dependencies requires a lot more validation. 

For instance, the version of go-systemd that is in use is from 2014, and there have been 242 commits, 11 releases, and several breaking changes since then, and that's just the one package.